fluid/DeepASR/data_utils/load_data.py

```diff
@@ -0,0 +1,183 @@
+#by zhxfl 2018.01.24
+""" @package docstring
+load speech data from disk
+"""
+
+import random
+import Queue
+import numpy
+import struct
+import data_utils.trans_mean_variance_norm as trans_mean_variance_norm
+import data_utils.trans_add_delta as trans_add_delta
+
+g_lblock = []
+g_que_sample = Queue.Queue()
+g_nframe_dim = 120 * 11
```

Comment:
Please don't use number directly, better to use const variable to tell the meaning or make it configurable.